tests/unittest/api_stability/references_committed/sampling_params.yaml

```diff
@@ -54,6 +54,9 @@ methods:
       min_p:
         annotation: Optional[float]
         default: null
+      beam_width_array:
```

Comment:
@wili-65535  The `references_committed` directory is for APIs committed for 1.0, we may need to move this new API into `references` directory instead.